OpenRA.Game/Actor.cs

```diff
@@ -280,29 +280,34 @@ public void Dispose()
 		}
 
 		// TODO: move elsewhere.
-		public void ChangeOwner(Player newOwner)
+		public void ChangeOwner(Player newOwner, World externalWorld = null)
```

Comment:
This change/parameter doesn't seem to be used anymore.

Reply:
Goddamm